Tesla.css

```diff
@@ -0,0 +1,233 @@
+body {
+    max-width: 1920px;
+    margin: 0 auto;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)